frontend/src/components/common/NameValueTable/NameValueTable.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import { Meta, Story } from '@storybook/react';
+import { Meta, StoryFn } from '@storybook/react';
```

Comment:
This and similar StoryFn changes could be done in a separate commit/PR.

Reply:
I think we should keep all storybook changes in the "Upgrade storybook setup" commit